pcweb/pages/docs/hosting/self_hosting.py

```diff
@@ -93,6 +93,14 @@ def self_hosting():
             pc.code("--backend-only"),
             " flags.",
         ),
+        doctext(
+            "It is also possible to export the components without zip it separate. To do this, use the ",
+            pc.code("--no-zip"),
+            " parameter.",
```

Comment:
Can we add a space after the `.` here. We need a way to catch this automatically as it's easy to miss.

Reply:
I have add the space